### PR TITLE
samples: matter: update versioning scheme for Matter applications

### PR DESCRIPTION
**Pull request by LuDuda**
This commit changes the default string representation of version to include the tweak.

### CODE REVIEW

**Review by wiba-nordic — APPROVED** [review]

**Comment by de-nordic** [comment]
@LuDuda West gone bad.

**Comment by jfischer-no** [comment]
Please rebase.

File: west.yml [hunk 1]
```diff
@@ -154,7 +154,7 @@ manifest:
     - name: matter
       repo-path: sdk-connectedhomeip
       path: modules/lib/matter
-      revision: e3dd453b9bd82ee33b258cd2be94abeca9ba62c0
+      revision: pull/395/head
```
Comment (jfischer-no) [inline west.yml:157]: 324ff7699e65c2120af6a6c665b6ccf63cda71d9, https://github.com/nrfconnect/sdk-connectedhomeip/commit/324ff7699e65c2120af6a6c665b6ccf63cda71d9

Comment (LuDuda) [reply west.yml]: @jfischer-no  can you double check if all is good (i hope) and merge?

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**